apps/trading/client-pages/market/market.tsx

```diff
@@ -132,7 +133,10 @@ export const MarketPage = () => {
         marketName={data?.tradableInstrument.instrument.name}
         decimalPlaces={data?.decimalPlaces}
       />
-      {tradeView}
+      <div className="flex flex-col h-full">
+        <MarketSuccessorBanner />
```

Comment:
We should keep market specific banners together. Can you move this to the trade grid and locate it next to the oracle banner?

Then we can also just pass market data as a prop which would simplify the banner logic a bit.

```
<div className="col-span-2 bg-vega-green">
        <MarketSuccessorBanner market={market} />
        <OracleBanner marketId={market?.id || ''} />
</div>
```

Reply:
done